app/src/main/res/navigation/nav_graph.xml

```diff
@@ -309,8 +309,19 @@
         <action
             android:id="@+id/action_bookmarkFragment_to_bookmarkAddFolderFragment"
             app:destination="@id/bookmarkAddFolderFragment" />
+        <action
+            android:id="@+id/action_global_bookmark_search_dialog"
```

Comment:
This isn't actually global since it's inside a fragment. It should follow a similar naming as `action_bookmarkFragment_to_bookmarkAddFolderFragment`. You can see examples of global nav near the top of the `nav_graph.xml`

Reply:
I see, I'll fix this.  Thanks